.github/workflows/schema.yaml

```diff
@@ -0,0 +1,45 @@
+name: schema
+
+on:
+  schedule:
+    - cron: 0 0 * * * # triggered nightly
+    # - cron: 0 0 * * 0 # triggered every sunday
```

Comment:
Needed?

Reply:
once its working ill switch it to weekly